lk2nd/lk2nd-device.c

```diff
@@ -275,6 +275,34 @@ static void lk2nd_parse_device_node(const void *fdt)
 #endif
 }
 
+struct lk2nd_keymap* lk2nd_fdt_parse_early_keys(void)
+{
+	int offset, len;
+	const uint32_t *val;
+	struct lk2nd_keymap *map = NULL;
+
+	void *fdt = (void*) lk_boot_args[2];
+	if (!fdt || dev_tree_check_header(fdt))
+		return NULL; // Will be reported later again. Hopefully.
+
+	offset = lk2nd_find_device_offset(fdt);
+	if (offset < 0)
+		return NULL;
+
+	val = fdt_getprop(fdt, offset, "lk2nd,keys", &len);
+	if (len > 0 && len % (3 * sizeof(uint32_t)) == 0) {
+		len /= (3 * sizeof(uint32_t));
+		/* sentinel: key=0 */
```

Comment:
What is this supposed to tell me?

Reply:
Probably a very bad attempt to say
` /* last element indicates end of the array with key=0 */ `